### DISCUSSION TITLE
Where are my projects saved? Recovery after Windows crash.

### DISCUSSION THREAD

**Discussion by ghost** [pybricks/support#1041] [Q&A]
Hello!
I created some projects on my old Win10 PC (Latest Pybricks installation local on my PC) which crashed. No I haven't made any data backups from my projects, but I have access to the harddrive and now search for my projects but can't find them.
In which path are the projects saved?
I want to import them now to my new Win11 system.

**Comment by dlech** [comment]
All of the "files" in Pybricks Code are stored in a database, so unfortunately, it is not as simple as finding the files on the hard drive.

You should be able to copy the user profile from your web browser on the old computer to the new computer to get your files back though. The browser user profile is in the `%LOCALAPPDATA%` folder either under `Google\Chrome\` or `Microsoft\Edge\` depending on which browser you use. Unless you created a custom profile, the profile name will be `Default.

So the full path is either `%LOCALAPPDATA%\Google\Chrome\User Data\Default` or `%LOCALAPPDATA%\Microsoft\Edge\User Data\Default`. You can either replace the existing user profile on your new computer (recommend backing up first) or rename the old profile and launch the browser using the old profile to get the old files, then switch back to the new profile.

**Comment by ghost** [comment]
Thanks, I found my projects again after simply copying Chrome's data to my new system.
